adobe_vipm/flows/validation/transfer.py

```diff
@@ -70,7 +70,10 @@ def get_prices(order, commitment, adobe_skus):
     product_id = order["agreement"]["product"]["id"]
     if (
         commitment
-        and commitment["status"] in (STATUS_3YC_COMMITTED, "ACTIVE")
+        and commitment["status"] in (
+            ThreeYearCommitmentStatus.COMMITTED,
```

Comment:
Here also group coul dbe created on the enum. Would be more concise and readable